x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/index.tsx

```diff
@@ -285,12 +285,6 @@ export const EditOutputFlyout: React.FunctionComponent<EditOutputFlyoutProps> =
               data-test-subj="settingsOutputsFlyout.nameInput"
               fullWidth
               {...inputs.nameInput.props}
-              placeholder={i18n.translate(
```

Comment:
Was the `placeholder` prop removed from `EuiFieldText` as well? (Just checking that this was not removed by mistake)

Reply:
Hi! Thank you, I double-checked and indeed looks like it was by mistake. I reverted